### ISSUE TITLE
Put an index.html page please

### ISSUE THREAD

**Issue by sskhandek** [ReflectionsProjections/rp2017#1]

**Comment by ananyacleetus** [comment]
same

**Comment by warut-vijit** [comment]
The Systems team is still waiting on the designer to get back to us with a layout. In the meantime, I can put up a placeholder if you want.

**Comment by sskhandek** [comment]
@warut-vijit Do we have an established font & colors & assets? We should put that in a README. I'll make a separate issue for that

**Comment by warut-vijit** [comment]
@sskhandek That will definitely be something that we'll work on this coming Tuesday.

**Comment by tyler-thetyrant** [comment]
@warut-vijit All the assets are available on RP drive.

**Comment by warut-vijit** [comment]
Also, I can't push directly to the repo, and can't submit a pull request/fork because it's empty at the moment

**Comment by sskhandek** [comment]
@warut-vijit I added the README.md & and index.html. Permission to close and open up a new issue regarding scaffolding, frameworks, and libraries?

**Comment by bluedrops** [comment]
r e s o l v e d b o y z
